dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/repositories/DataRequestRepository.kt

```diff
@@ -26,15 +24,13 @@ interface DataRequestRepository : JpaRepository<DataRequestEntity, String> {
      * @param datalandCompanyId to check for
      * @param dataType to check for
      * @param reportingPeriod to check for
-     * @param requestStatus to check for
      * @returns a list of the data requests
      */
-    fun findByUserIdAndDatalandCompanyIdAndDataTypeAndReportingPeriodAndRequestStatus(
+    fun findByUserIdAndDatalandCompanyIdAndDataTypeAndReportingPeriod(
```

Comment:
you can probably delete the words "and" from the function name. the meaning will still be clear i guess

Reply:
I guess you cant. I think the DataRequestRepository creates the sql query based on the name! Since there is no request Status anymore, well filter the resulting list (its a small list only for one user and one company)

Reply:
correct @jhaerDfine